src/vs/workbench/contrib/searchEditor/browser/searchEditor.contribution.ts

```diff
@@ -538,3 +540,36 @@ registerAction2(class OpenSearchEditorAction extends Action2 {
 	}
 });
 //#endregion
+
+//#region Search Editor Working Copy Editor Handler
+class SearchEditorWorkingCopyEditorHandler extends Disposable implements IWorkbenchContribution {
+
+	constructor(
+		@IInstantiationService private readonly instantiationService: IInstantiationService,
+		@IWorkingCopyEditorService private readonly workingCopyEditorService: IWorkingCopyEditorService,
+	) {
+		super();
+
+		this.installHandler();
+	}
+
+	private installHandler(): void {
+		this._register(this.workingCopyEditorService.registerHandler({
+			handles: async workingCopy => {
+				return workingCopy.resource.scheme === SearchEditorConstants.SearchEditorScheme;
```

Comment:
My understanding is that any backup for search editors is using the `search-editor` scheme, independent of the search being saved or not.

Reply:
Yes, I believe this is correct